device-types/Supermicro/SYS-1019P-WTR.yaml

```diff
@@ -0,0 +1,71 @@
+---
+manufacturer: Supermicro
+model: SYS-1019P-WTR
+slug: sys-1019-wtr
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+comments: 'Specification: [https://www.supermicro.com/en/products/system/1U/1019/SYS-1019P-WTR.cfm](https://www.supermicro.com/en/products/system/1U/1019/SYS-1019P-WTR.cfm)'
+console-ports:
+  - name: COM1
+    type: de-9
+    label: Rear
+
+  - name: COM2
+    type: de-9
+    description: Header
+  - name: Front VGA
+    type: other
```

Comment:
Questionable as to whether this counts as a console port

Reply:
Personally, I would remove USB interfaces all together.  Let the end-consumer decide if they want to model them.  I think there is a use case for having the USB ports, but I think it requires different handling as USB ports can be different things.

For example, on catalyst switches, there is a USB port on the back that can accept a 120G SSD, that would be best modelled as a inventory item labelled "USBFLASH1" or similar.

You can leave the VGA port as a console because it is a type of console but I suspect many people are not connecting into it.  This is the same as with the USB.  For example, lets say you want to model your KVM device, you might want those as a single "console-server-port" on the KVM but it spiders out and connects to both a monitor and USB port on the server.

TLDR;

Leave the VGA

Ditch the USB - The end user can model it how they desire

Reply:
Corrected in https://github.com/netbox-community/devicetype-library/commit/44d267b3c9d05a6b202660c02221d13d5549e9a0